network/config_test.go

```diff
@@ -15,3 +15,19 @@ func TestConf(t *testing.T) {
 	conf.EnableRelay = false
 	assert.NoError(t, conf.SanityCheck())
 }
+
+func TestValidateAddress(t *testing.T) {
```

Comment:
It's not recommended to test unreported functions. 
So, move these tests into  `TestDefaultConfigCheck` (rename `TestConf` to `TestDefaultConfigCheck`.